app/src/bcsc-theme/features/verify/VerificationMethodSelectionScreen.tsx

```
@@ -34,17 +36,35 @@ const VerificationMethodSelectionScreen = ({ navigation }: VerificationMethodSel
   const handlePressSendVideo = useCallback(async () => {
     try {
       setSendVideoLoading(true)
+
       const { sha256, id, prompts } = await evidence.createVerificationRequest()
+
+      await Promise.all([
```

Question: would it better to use Promise.allSettled so we can do as much as possible and see if more than one of the promises failed, rather than fail early if any fail?

---

Yea I think that would probably be a better approach.

Changed.